js/interface.js

```diff
@@ -2383,11 +2383,15 @@ function getSecurityRule() {
 
   if (currentFinalRules.length > 0) {
     currentFinalRules.forEach(function(rule) {
-      if (widgetData.tokenId) {
-        if (rule.allow.hasOwnProperty('tokens') && Number(rule.allow.tokens[0]) === widgetData.tokenId) {
+      if (!rule.allow.hasOwnProperty('tokens')) {
```

Comment:
See suggestion below with `_.some()`. You won't need to check this first, and you can also shorten the loop

```suggestion
```

Reply:
Agreed, Done changes